.gitignore

```diff
@@ -11,3 +11,4 @@ latest.dump
 node_modules/
 public
 yarn-error.log
+.ruby-version
```

Comment:
`.ruby-version` was removed from this repo when switching to asdf. I still use rbenv though and find it useful to have `.ruby-version` in the repo.